### PR TITLE
Case Study part 1 (completed)

### PR DESCRIPTION
**Pull request by imdjsarge**

### CODE REVIEW

File: yurts.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <dl>
+        <h3>What is a yurt?</h3>
+        <p>Our luxury yurts are permanent structures four feet off the ground.  Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+        <h3>How are the yurts furnished?</h3>
+        <p>Each yurt is furnished with a queen-sized bed with down quilt and gas-fired stove. The luxary camping experience also included electricity and a sink with hot and cold running water.  Shower and restroom facilities are located int he lodge.</p>
+        <h3>What should I bring?</h3>
+        <p>Bring a sense of adventure and some time to relax!  Most guests also packt comfortable walking shoes and plan to dress for changing weather with layers of clothing.</p>
+    </dl>
+    </div>
+    <footer><p><small>Copyright &copy; 2015 Pacific Trails Resort</small><br><small><a href="mailto:rgallo@my.hennepintech.edu">rgallo@my.hennepintech.edu</a></small></p></footer>
```
Comment (htcMosman) [inline yurts.html]: We should not use the <small> tags, as this is better done via CSS.

File: yurts.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </nav>
+    <div>
+        <h3>The Yurts at Pacific Trails</h3>
+    <dl>
+        <h3>What is a yurt?</h3>
+        <p>Our luxury yurts are permanent structures four feet off the ground.  Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+        <h3>How are the yurts furnished?</h3>
+        <p>Each yurt is furnished with a queen-sized bed with down quilt and gas-fired stove. The luxary camping experience also included electricity and a sink with hot and cold running water.  Shower and restroom facilities are located int he lodge.</p>
+        <h3>What should I bring?</h3>
+        <p>Bring a sense of adventure and some time to relax!  Most guests also packt comfortable walking shoes and plan to dress for changing weather with layers of clothing.</p>
```
Comment (htcMosman) [inline yurts.html]: Typos! line 24 "packt".  Line 23  "int he".  Line 22 "luxary".
Business customers aren't happy with typos in their content.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Yurts</title>
+    <link rel="stylesheet" href="pacific.css" />
+    <meta charset"utf-g" />
```
Comment (htcMosman) [inline yurts.html]: The charset should be utf-8.

File: reservations.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <titlePacific Trails</title>
+    <link rel="stylesheet" href="pacific.css" />
+    <meta charset"utf-g" />
```
Comment (htcMosman) [inline reservations.html]: The charset should be utf-8.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Pacific Trails</title>
+    <link rel="stylesheet" href="pacific.css" />
+    <meta charset"utf-g" />
```
Comment (htcMosman) [inline index.html]: The charset should be utf-8.

File: activities.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <titlePacific Trails</title>
+    <link rel="stylesheet" href="pacific.css" />
+    <meta charset"utf-g" />
```
Comment (htcMosman) [inline activities.html]: The charset should be utf-8.

File: activities.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <titlePacific Trails</title>
```
Comment (htcMosman) [inline activities.html]: Missing the `>` on the `<title>` tag.  Make sure to validate to catch errors like this.  Validation errors are a large point deduction.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <div>
+        <h2>Enjoy Nature in Luxury</h2>
+        <p>Pacific Trails Resort offers a special lodging experiences on the California North Coast.  Relax in serenity with panoramic views of the Pacific Ocean.</p>
+        <ul>
+            <li>Private yurts with decks overlooking the ocean</li>
+            <li>Activities lodge with fireplace and gift shop</li>
+            <li>Nightly fine dining at the Overlook Cafe</li>
+            <li>Heated outdoor ppl and whirlpool</li>
```
Comment (htcMosman) [inline index.html]: Typo!  "ppl"

File: index.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+            <li>guided hiking tours of the redwoods</li>
```
Comment (htcMosman) [inline index.html]: First letter should be capitalized.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+        </ul>
+        <p>Pacific Trails Resort<br>
```
Comment (htcMosman) [inline index.html]: Address should be in a <div id="contact">.
Make sure that you are working on the assignments from the website, not the textbook.

File: reservations.html [hunk 11]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <titlePacific Trails</title>
```
Comment (htcMosman) [inline reservations.html]: Missing the `>` on the `<title>` tag.  Make sure to validate to catch errors like this.  Validation errors are a large point deduction.

File: yurts.html [hunk 12]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Yurts</title>
```
Comment (htcMosman) [inline yurts.html]: The title should be "Pacific Trails :: Yurts"

File: reservations.html [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+</head>
+    <body>
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <h1><strong>Page is under construction!<strong><h1>
```
Comment (htcMosman) [inline reservations.html]: The file should still end with the `</body>` and the `</html> tag.  You might also like to include the page footer.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+        12010 Pacific Trails Road<br>
+        Zephyr, CA 95555<br><br>888-555-5555<br></p>
+        </div>
+        <footer><p><small>Copyright &copy; 2015 Pacific Trails Resort</small><br><small><a href="mailto:rgallo@my.hennepintech.edu">rgallo@my.hennepintech.edu</a></small></p></footer>
```
Comment (htcMosman) [inline index.html]: We should not use the `<small>` tags, as this is better done via CSS.

File: activities.html [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+</head>
+    <body>
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <h1><strong>Page is under construction!<strong><h1>
```
Comment (htcMosman) [inline activities.html]: The file should still end with the `</body>` and the `</html> tag.  You might also like to include the page footer.

File: reservations.html (hunk 13 above)
Comment (htcMosman) [inline reservations.html]: The file should still end with the `</body>` and the `</html> tag.  You might also like to include the page footer.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+    </body>
```
Comment (htcMosman) [inline index.html]: File should end with the `</html>` tag.

File: index.html [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <div>
+        <h2>Enjoy Nature in Luxury</h2>
+        <p>Pacific Trails Resort offers a special lodging experiences on the California North Coast.  Relax in serenity with panoramic views of the Pacific Ocean.</p>
```
Comment (htcMosman) [inline index.html]: Missing the "resort" style to call out the company name.  Use here and in the contact information below.

File: pacific.css [hunk 18]
```diff
@@ -0,0 +1,17 @@
...
+p {
+    color: #666666;
+}
+nav {
+    background-color: #90C7E3;
+}
+h2 { color: #3399CC;
+}
+h3 {color: #000033;
+}
```
Comment (htcMosman) [inline pacific.css]: The resort style is missing.

File: yurts.html [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+    <body>
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>     
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <div>
+        <h3>The Yurts at Pacific Trails</h3>
```
Comment (htcMosman) [inline yurts.html]: This should be an `<h2>`, but the ones below are correctly `<h3>` tags.

File: yurts.html [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
+    <dl>
```
Comment (htcMosman) [inline yurts.html]: The `<dl>` was from the book instructions... we must have missed this last week cleaning up.  Make sure to follow the instructions from the website, not the book in the future.

File: yurts.html [hunk 21]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+    <body>
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>     
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <div>
```
Comment (htcMosman) [inline yurts.html]: The `<div>` tag should have an id="content" attribute.

File: index.html [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+</head>
+    <body>
+        <header><h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <a href="index.html">Home</a> &nbsp;
+        <a href="yurts.html">Yurts</a> &nbsp;
+        <a href="activities.html">Activities</a> &nbsp;
+        <a href="reservations.html">Reservations</a> &nbsp;
+    </nav>
+    <div>
```
Comment (htcMosman) [inline index.html]: Putting the "under construction" in the href gives a validation error.  Often what people will do when there isn't a page yet is use # in the href.  That just causes the link to go to the current page or seem like it does nothing.

Comment (imdjsarge) [inline index.html]: Hi Mary.  I can’t see to find how to get rid of the hyperlinks in the <nav>.  I also am confused about your comment below regarding the reservations page. 

 

So far everything else looks great.  



Sarge

**Pull request closed without merging**